net-ftp/tnftp/tnftp-20151004.recipe

```diff
@@ -1,14 +1,16 @@
 SUMMARY="NetBSD luke FTP client"
 DESCRIPTION="NetBSD FTP client with several advanced features"
```

Comment:
"You could elaborate the DESCRIPTION with what's at http://freecode.com/projects/tnftp/:

tnftp (formerly known as lukemftp) is a port of the NetBSD FTP client. It offers many enhancements over the traditional BSD FTP client, including
* command-line editing,
* command-line fetches of FTP and HTTP URLs (including via proxies),
* command-line uploads of FTP URLs,"

... etc

Mind the 80 char limit (compare to other recipes)